### ISSUE TITLE
Notice: Undefined index: unique_key in /stcr_options.php on line 61

### ISSUE THREAD

**Issue by IEatApples** [stcr/subscribe-to-comments-reloaded#680]
I'm getting a series of errors:

**On the Plugin Settings page:**
Notice: Undefined index: unique_key in /wp-content/plugins/subscribe-to-comments-reloaded/options/stcr_options.php on line 61

**And on the frontend after clicking Manage Subscription:**
Notice: Trying to get property 'ID' of non-object in /home/wp-includes/class-wp-query.php on line 4121
Notice: Trying to get property 'post_title' of non-object in /home/wp-includes/class-wp-query.php on line 4123
Notice: Trying to get property 'post_name' of non-object in /home/wp-includes/class-wp-query.php on line 4125

class="archive category logged-in admin-bar no-customize-support wp-custom-logo wp-embed-responsive sticky-menu-fade sticky-enabled both-sticky-menu mobile-header mobile-header-logo mobile-header-sticky right-sidebar nav-below-header separate-containers nav-search-enabled header-aligned-left dropdown-hover" itemtype="https://schema.org/Blog" itemscope>

**Comment by BobaWebDev** [comment]
Thanks for reporting, we'll test that out.

Do you have **WP_DEBUG** set to **true** in wp-config? Notices shouldn't show up unless in DEBUG mode.

**Comment by IEatApples** [comment]
Yes, I inadvertently had WP_DEBUG set to true, so that's why I thought there was an error. So about that. Thanks!

**Comment by BobaWebDev** [comment]
You're welcome.
